docs/components/position.md

```diff
@@ -45,3 +45,38 @@ entity. In the local parent's space, `#child1`'s position would be `0 0 0`.
 The world-space position of `#child2` would be `3 5 7`, by combining the
 position with the parent entity. In the parent's local space, `#child2`'s
 position would be `2 3 4`.
+
+## Updating Position
+
+[update]: ../introduction/javascript-events-dom-apis.md#updating-a-component-with-setattribute
+[vector]: https://threejs.org/docs/index.html#api/math/Vector3
+
+For performance and ergonomics, we recommend updating position directly via the
+three.js `Object3D.position` Vector3 versus [via `.setAttribute`][update].
+
+This method is easier because we have access to all the [Vector3
+utilities][vector], and faster by skipping `.setAttribute` overhead and not
+needing to create an object to set rotation:
+
+```js
+// With three.js
+el.object3D.position.set(1, 2, 3);
+
+// With .setAttribute (less recommended).
+el.setAttribute('position', {x: 1, y: 2, z: 3});
+```
+
+Also easier to do incremental updates:
+
+```js
+// With three.js
+el.object3D.position.x += 1;
+
+// With .setAttribute (less recommended).
+var position = el.getAttribute('position');
+position.x += 1;
+el.setAttribute('position', position);
```

Comment:
Isn't the `setAttribute` here superfluous since `getAttribute` now returns the actual `object3D.position` `Vector3`?

Reply:
Good catch